src/components/Answers/AnswerForm.js

```diff
@@ -55,7 +55,7 @@ function AnswerForm(props) {
 
         <AnswerWrapper >   
             <form className="answer-form" onSubmit={handleSubmit}>
-                <textarea name="answer" placeholder="Answer" value={newAnswer.answer} onChange={handleChange}  />
+                <textarea name="answer" rows="4" cols="50" placeholder="Answer" value={newAnswer.answer} onChange={handleChange}  />
```

Comment:
Hey I completely re-did AnswerForm to use Formik. check it out and let me know if you want to keep these changes or no.